client/src/features/app/app.component.js

```diff
@@ -1,18 +1,16 @@
-import React, { Component } from 'react';
-import { ListItem } from '../list-item/list-item.component';
-import logo from '../../logo.svg';
-import apiService from '../../shared/services/api-service';
-import { SearchBarContainer } from '../search-bar/search-bar.container';
-import { AutoCompleteListContainer } from '../auto-complete-list/auto-complete-list.container';
-import { LocationRatingContainer } from '../location-rating/location-rating.container';
-import { GoogleMap } from "../google-map/google-map.component";
+import React, { Component } from "react";
+import apiService from "../../shared/services/api-service";
+import { SearchBarContainer } from "../search-bar/search-bar.container";
+import { AutoCompleteListContainer } from "../auto-complete-list/auto-complete-list.container";
+import { LocationRatingContainer } from "../location-rating/location-rating.container";
+import { Header } from "../../ui-kit/header";
 
```

Comment:
Why is the GoogleMap component removed?

Reply:
I think it was because googlemaps was causing circle ci tests to fail and Katia said she will add it back in on her new pull request!